subworkflows/local/infer_trees.nf

```diff
@@ -0,0 +1,31 @@
+include { CIALIGN as TRIM_MSAS } from '../../modules/local/cialign'
+include { FASTTREE             } from '../../modules/local/fasttree'
+include { WITCH as ALIGN_SEQS  } from '../../modules/local/witch'
```

Comment:
Hmmm.... I thought in one of my previous commits i overrode `WITCH as ALIGN_SEQS` with `FAMSA as ALIGN_SEQS`...